docs/src/main/sphinx/connector/iceberg.rst

```diff
@@ -141,6 +141,7 @@ When using any database besides PostgreSQL, a JDBC driver jar file must be place
     connector.name=iceberg
     iceberg.catalog.type=jdbc
     iceberg.jdbc-catalog.catalog-name=test
+    iceberg.jdbc-catalog.driver-class=org.postgresql.Driver
```

Comment:
Do we need to include driver-class property in the minimum requirenments?

```
At a minimum, ``iceberg.jdbc-catalog.connection-url`` and
``iceberg.jdbc-catalog.catalog-name`` must be configured.
```
https://github.com/trinodb/trino/pull/16196/commits/4400fc0ef7e044187986bdf15c66f7005bd07ed7#diff-e1aabf1cfd8bd8aa7d1b75e70089b57413b2e620a5eebeb36cc76fd3f2ac60dbR135-R136